build/deprecated/build-utils/Cargo.toml

```diff
@@ -11,4 +11,5 @@ serde = { version = "1.0", features = ["derive"] }
 
 [dependencies.reqwest]
```

Comment:
@mwu-tow it looks like this crate might not even use reqwest anymore? Perhaps this dep (and the others) could be removed entirely?

Reply:
@indiv0 I dont think it is unused. I see a number of usages, and this is a transitive dependency of other crates.
Bug you can always try. In general if you can remove something and everything still builds, then go ahead.